README.md

```diff
@@ -33,13 +33,13 @@ Proxy-Wasm extensions across different proxies.
 * [Envoy]
 * [Istio Proxy] (Envoy-based)
 * [MOSN]
+* [ATS]
 * [OpenResty] (work-in-progress)
-* [ATS] (proof-of-concept)
 
 [Envoy]: https://github.com/envoyproxy/envoy
 [Istio Proxy]: https://github.com/istio/proxy
 [MOSN]: https://github.com/mosn/mosn
-[ATS]: https://github.com/jplevyak/trafficserver/tree/wasm
+[ATS]: https://github.com/apache/trafficserver/tree/master/plugins/experimental/wasm
```

Comment:
I think that [docs](https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/wasm.en.html) might be a better link.

Reply:
I think so, too. But it is then different from the others, which are links to git repo. 
Is that ok? If you say so, i will make the change.

Reply:
Others lack the proper documentation, or at least they did at the time.